gst/nnstreamer/nnstreamer_plugin_api_impl.c

```diff
@@ -1557,8 +1557,9 @@ gst_tensor_buffer_get_nth_memory (GstBuffer * buffer, const guint index)
   }
 
   num_tensors = gst_tensor_buffer_get_count (buffer);
-  if (num_tensors == 0U) {
-    nns_loge ("num_tensors is 0. Please check the buffer.");
+  if (index >= num_tensors) {
```

Comment:
Since ```index``` starts from 0, it seems that an exception is handled if the return value of ``` gst_tensor_buffer_get_count()``` is 0.